packages/oas-utils/src/entities/spec/spec-objects.ts

```diff
@@ -57,7 +57,7 @@ export const oasInfoSchema = z.object({
    * REQUIRED. The version of the OpenAPI document (which is distinct from the OpenAPI
    * Specification version or the API implementation version).
    */
-  version: z.string().default('0.0.1'),
+  version: z.string().optional().default('0.0.1'),
```

Comment:
Not related to this PR, but maybe a great opportunity to clean this up:

Semantic versions start at 0.1.0, not 0.0.1. But I think the default should just be `null`. I don’t see a reason to make up a version number if the users doesn’t add one. Wdyt?

Reply:
I disagree just because the spec says it is required, so one should always exist. The integrations should just pull it from the package.json

Reply:
What package.json? 🤔 This is for whatever OpenAPI document the user gives us, right? There’s no package.json involved typically.

Let’s use 1.0 as the default then, though. Feels like a better fallback.

```suggestion
  version: z.string().optional().default('1.0'),
```